src-java/me/tonsky/persistent_sorted_set/Loader.java

```diff
@@ -0,0 +1,7 @@
+package me.tonsky.persistent_sorted_set;
+import java.util.UUID;
+
+public interface Loader {
```

Comment:
Since it does both load and save, maybe rename to Storage?

Reply:
Done.